x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/NodeDeprecationChecks.java

```diff
@@ -169,6 +169,17 @@ static DeprecationIssue watcherNotificationsSecureSettingsCheck(Settings nodeSet
         return null;
     }
 
+    static DeprecationIssue watcherHipchatNotificationSettingsCheck(Settings nodeSettings, PluginsAndModules plugins) {
+        if (nodeSettings.getByPrefix("xpack.notification.hipchat.").size() > 0) {
+            return new DeprecationIssue(DeprecationIssue.Level.CRITICAL,
```

Comment:
I think this should be a warning since it only possibly an issue. @gwbrown - thoughts on critical vs. warning ?

Reply:
Depends on how 7.0 behaves if you leave this setting in place. Most removed node settings are `CRITICAL` because Elasticsearch will refuse to start if you have unrecognized settings in `elasticsearch.yml`.  I think that's the case here, so `CRITICAL` is the correct level, although I could be wrong if Watcher is doing something funny with prefix settings.